nsd-control.8.in

```diff
@@ -35,8 +35,11 @@ There are several commands that the server understands.
 .TP
 .B start
 Start the server. Simply execs \fInsd\fR(8).  The nsd executable 
-is searched for in the \fBPATH\fR set in the environment.  It is started 
-with the config file specified using \fI\-c\fR or the default config file.
+is not searched for in the \fBPATH\fR set in the environment.  Instead the
+default location relative to the installation prefix specified at
+compile-time.  The executable location can be overridden by setting
+\fINSD_PATH in the environment.  It is started with the config file
+specified using \fI\-c\fR or the default config file.
```

Comment:
You need to en the `\fI` on NSD_PATH with `\fR` after NSD_PATH.  Github renders the italics on the manpage nroff, by the way.

Reply:
done